web/cypress/integration/hfi-calculator-page.spec.ts

```diff
@@ -11,18 +11,21 @@ describe('HFI Calculator Page', () => {
 
     it('should display Daily View Table after landing on page', () => {
       cy.visit(HFI_CALC_ROUTE)
+      cy.get(`[aria-label="daily toggle"]`).click()
```

Comment:
We should be using `data-testid` attributes like everywhere else, `aria-labels` may change.

Reply:
I was using a test id and it kept failing, even though it was in the same place in the test as this get statement. I triple checked the spelling as well.

Reply:
Let me see if I can try it again

Reply:
Can you post what you tried?